modules/ROOT/pages/6.4.2-release-notes.adoc

```diff
@@ -76,6 +76,13 @@ include::partial$misc/admon-releasenotes-for-stable.adoc[]
 
 // {productname} 6.4.2 also includes the following bug fixes:
 
+=== Selection was not correctly scrolled horizontally into view when using the `selection.scrollIntoView` API.
+
+In previous versions of {productname}, the function `scrollToMarker` calculated how much should be scrolled vertically, but the calculation for for the horizontal scroll, was set to the position `left` of the marker.
+
+As a consequence of this, in some situations the horizontal scroll would not scroll to the correct position.
+
+To fix this, instead of manually calculating the vertical and horizontal position, {productname} 6.4.2 now uses a native `scrollIntoView` to move the scroll to the marker.
```

Comment:
```suggestion
In previous versions of {productname}, the `scrollToMarker` function manually calculated how much should be scrolled vertically. The horizontal scroll amount, however, was set to a specific value: left of the marker.

As a consequence, in some circumstances {productname} editor instances did not scroll to the correct horizontal position.

To fix this, instead of manually calculating the vertical and horizontal marker position, {productname} 6.4.2 now uses a native `scrollIntoView` function to keep the marker in the viewport.
```

Copy edits.